.gitignore

```diff
@@ -1,10 +1,9 @@
 *.iml
 .gradle
-/local.properties
-/.idea/workspace.xml
-/.idea/libraries
+local.properties
 .DS_Store
-/build
-/captures
+build/
+captures/
 .idea
-
+app/debug/
+app/release/
```

Comment:
Hi. Thanks for the PR. Just off to get married, so will be a while before I get a chance to look at this.

First issue: app/release is supposed to be in the repo. The apk is provided here for users unable to access the Play Store.

Reply:
It would be good to use tags for releases. You can create a release on GitHub anytime a new version is released and attach the release APK to it. That is generally a much more accepted way of distribution. See [NewPipe](https://github.com/TeamNewPipe/NewPipe/releases) for an example.

Also, congrats!

Reply:
Thanks. Sorry, still been too busy to look at this properly with wedding and new job. Looks mainly very useful though. Hopefully will be able to merge it shortly.

Yes, please revert the gitignore change. I'm sure there's a better way to do it (thanks for the example - could well be useful to do it that way) but it's what we have for now.